src/main/java/net/yogstation/yogbot/listeners/commands/MActivityCommand.kt

```diff
@@ -0,0 +1,84 @@
+package net.yogstation.yogbot.listeners.commands
+
+import discord4j.core.event.domain.message.MessageCreateEvent
+import net.yogstation.yogbot.DatabaseManager
+import net.yogstation.yogbot.config.DiscordConfig
+import net.yogstation.yogbot.permissions.PermissionsManager
+import net.yogstation.yogbot.util.DiscordUtil
+import net.yogstation.yogbot.util.StringUtils
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+import java.util.*
+
+@Component
+class MActivityCommand(discordConfig: DiscordConfig,
+					   permissions: PermissionsManager,
+					   private val databaseManager: DatabaseManager
+) : PermissionsCommand(discordConfig, permissions) {
+	override val name = "mactivity"
+	override val requiredPermissions = "mhelp"
+
+	override fun doCommand(event: MessageCreateEvent): Mono<*> {
+		return databaseManager.byondDbConnection.use { connection ->
+			connection.createStatement().use { stmt ->
+				val activities: MutableMap<String, Double> = HashMap()
+				var mentorLen = 8
+				stmt.executeQuery("SELECT ckey FROM ${databaseManager.prefix("mentor")}").use { results ->
```

Comment:
Why not use JPA here?

Reply:
Not sure how it interacts with prefixes, and I don't want it messing up the tables